src/Symfony/Bridge/PhpUnit/bootstrap.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Bridge\PhpUnit\DeprecationErrorHandler;
+
+if (!class_exists('PHPUnit_Util_ErrorHandler')) {
+    return;
+}
+
+if (PHP_VERSION_ID >= 50400 && gc_enabled()) {
+    // Disabling Zend Garbage Collection to prevent segfaults with PHP5.4+
+    // https://bugs.php.net/bug.php?id=53976
+    gc_disable();
+}
+
+DeprecationErrorHandler::register('strict' === getenv('SYMFONY_DEPRECATIONS'));
+
+if (class_exists('Doctrine\Common\Annotations\AnnotationRegistry')) {
+    AnnotationRegistry::registerLoader('class_exists');
+}
```

Comment:
not needed. The MongoDB annotations are autoloadable, so this is enough to register them (registering class_exists in the AnnotationRegistry is enough as long as all your autoloaders are handling properly the case of a non-existant class, i.e. not breaking ``class_exists``)